modules/rosa-cluster-autoscaler-cli-settings.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+// * osd_cluster_admin/osd-cluster-autoscaling.adoc
+
+:_content-type: REFERENCE
+[id="rosa-cluster-cli-autoscale-settings_{context}"]
+= Cluster autoscaling settings using the ROSA CLI
+
+There are several options you can add to the cluster creation command to adjust autoscaling options. The tables explain all the configurable autoscaler settings when using the ROSA CLI (`rosa`).
+
+[cols="4",options="header"]
+|===
+|Setting
+|Description
+|Type or Range
+|Example/Instruction
+
+|`--autoscaler-balance-similar-node-groups`
+|Identify node groups with the same instance type and label set, and aim to balance respective sizes of those node groups.
+|`boolean`
+|Add it to set to true, do not add to set to false.
+
+|`--autoscaler-skip-nodes-with-local-storage`
+|If set, cluster autoscaler will never delete nodes with pods with local storage, for example, EmptyDir or HostPat.
+|`boolean`
+|Add it to set to true, do not add to set to false.
+
+|`--autoscaler-log-verbosity _int_`
```

Comment:
I think this column only needs to write `--autoscaler-log-verbosity` no need to write `int` because there is a column of "Type or Range" behind it, the same for the other parameters.